src/capi.zig

```diff
@@ -122,6 +122,7 @@ fn errorToInt(err: Error) i32 {
         Error.NegativeErrorBound => return 3,
         Error.IncorrectInput => return 4,
         Error.OutOfMemory => return 5,
+        Error.EmptySet => return 6,
```

Comment:
Maybe rename to `Empty` so it can be used by any custom collection implemented by the library.

Reply:
By using ArrayList for handling the upper and lower hull I dont need to define my own error anymore. I remove this.